utils/test-cypress.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+set -e
+
+PORT=25444
+default_bitcoind_datadir=/tmp/specter_btcd_regtest_plain_datadir
+
+
+. ./.env/bin/activate
+
+function sub_default {
+    echo "Usage: ./utils/test-cypress.sh <subcommand> [options]"
+    echo "  Doing stuff with cypress-tests according to <subcommand>"
+    echo ""
+    echo "Subcommands:"
+    echo "    open                  will open the cypress app."
+    echo "    run                   will run the tests."
+    echo "    snapshot <spec_file>  will create a snapshot of the spec-file. It will create a tarball"
+    echo "                          of the btc-dir and the specter-dir and store that file in the "
+    echo "                          ./cypress/fixtures directory"
+    echo ""
+    echo "open and run take a spec-file optionally. If you add a spec-file, then automatically the"
+    echo "corresponding snapshot is untarred before and, in the case of run, only the spec-file and"
+    echo "all subsequent spec_files are executed."
+    echo ""
+}
+
+function start_bitcoind {
+    echo "Startting bitcoind ..."
+    if [ "$1" == "reset" ]; then
+        if [ "$DOCKER" != "true" ]; then
+            python3 -m cryptoadvance.specter bitcoind --reset --nodocker --config CypressTestConfig
+        fi
+    fi
+    if [ "$DOCKER" != "true" ]; then
+        addopts=--nodocker
+    fi
+    python3 -m cryptoadvance.specter bitcoind $addopts --create-conn-json --config CypressTestConfig &
+    bitcoind_pid=$!
+}
+
+function stop_bitcoind {
+    if [ ! -z ${bitcoind_pid+x} ]; then
+        kill $bitcoind_pid
```

Comment:
This seems to fail for me for some reason, causing the stop_specter command to not execute. (if I switch their order at least specter stops properly)


I think it does stop the python process, but not the bitcoind itself...

Reply:
should be fixed with 6e2dcca